src/comment.ts

```diff
@@ -59,9 +59,11 @@ export function renderComment({
   // Build footer
   let footer = ''
   if (includeFooter === undefined || includeFooter === true) {
-    footer =
-      `\n\n---\n\n_Triggered by @${github.context.actor},` +
-      ` Commit: \`${(github.context.payload as PullRequestEvent).pull_request.head.sha}\`_`
+    let commitInfo = ''
+    if ('pull_request' in github.context.payload) {
+      commitInfo = `, Commit: \`${(github.context.payload as PullRequestEvent).pull_request.head.sha}\``
+    }
+    footer = `\n\n---\n\n_Triggered by @${github.context.actor}${commitInfo}_`
```

Comment:
[nit] I'd just modify the `footer` in the `if` block above, i.e. run this before the if condition, then conditionally add to the footer. I was a little confused when reading through it just now.